modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.js

```diff
@@ -29,7 +30,11 @@ const Cards = ({items, schema}) => {
 							className="col-md-3"
 							key={item[selectedItemsKey] || index}
 						>
-							<Card item={item} schema={schema} />
+							<Card
+								item={item}
+								schema={schema}
+								spritemap={SPRITEMAP}
```

Comment:
`spritemap` should not be needed, we provide it in context for Clay components. Not sure why this is not working in this specific component.

Reply:
Should I remove it?